scripts/set_deployment_image_tag.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Generic script that sets the image+tag of a container in a deployment's pod spec in user namespaces
+# As is.  Will set deployment's "r-studio-server" container's image tag to "v1.3.2" in all user namespaces
+
+set -e
+
+DRY_RUN=true
+
+while true; do
```

Comment:
Fancy dialog, uh :P?

Reply:
not saying change it but have you seen the `dialog` command? 
`dialog --yesno "Is this a dry run?" 6 25; echo $?`

Reply:
✨  - didn't know it. curses galore :D 

I wonder how that would work in a non-interactive run, I played with [`--defaultno --sleep 5`](https://linux.die.net/man/1/dialog) but it doesn't seem to do what I would expect (default to "No" after 5 seconds unless I select something else before) 🤷‍♂️  - nice trick tho.

Reply:
yeah it's a bit too fancy, I actually prefer this `while true` but I guess you can do `--timeout 5` instead of sleep, but timeout always exits with code 255